### PR TITLE
Resolve ErrorProne warnings

### PR DESCRIPTION
**Pull request by dmdashenkov**
Since a recent PR, we've been observing a number of code style warnings from ErrorProne. This PR resolves those warnings.

### CODE REVIEW

**Comment by dmdashenkov** [comment]
@alexander-yevsyukov, PTAL. @armiol, PTAL. @serhii-lekariev, FYI.

File: base/src/main/java/io/spine/base/Tests.java [hunk 1]
```diff
@@ -71,7 +73,7 @@
      *         the system property explicitly.
      */
     @Override
-    protected boolean enabled() {
+    public boolean enabled() {
```
Comment (armiol) [inline base/src/main/java/io/spine/base/Tests.java:76]: This was a design purpose to prohibit invoking this method from outside the package. This is why they were not enums.

**Comment by serhii-lekariev** [comment]
I propose https://github.com/SpineEventEngine/base/commit/bb9f27675e1f0cffd995977a98497c969ff5c945 and https://github.com/SpineEventEngine/base/commit/82fd14d819810a9c5e8a9f5f4f833ba80240eed2, which seem to resolve the warnings. As for `EnvironmentType` design - let's discuss.

**Comment by serhii-lekariev** [comment]
As discussed with @dmdashenkov, I'll slightly redesign `EnvironmentType` in https://github.com/SpineEventEngine/base/pull/542. I'll make sure to deal with the Error Prone warnings too.

**Comment by dmdashenkov** [comment]
@serhii-lekariev, I'll close the PR but leave the branch alive for now.

**Pull request closed without merging**